pkg/kubelet/volumemanager/volume_manager_test.go

```diff
@@ -244,8 +231,8 @@ func TestInitialPendingVolumesForPodAndGetVolumesInUse(t *testing.T) {
 	// delayed claim binding
 	go delayClaimBecomesBound(kubeClient, claim.GetNamespace(), claim.ObjectMeta.Name)
 
-	err = wait.Poll(100*time.Millisecond, 1*time.Second, func() (bool, error) {
-		err = manager.WaitForAttachAndMount(context.Background(), pod)
+	err := wait.Poll(100*time.Millisecond, 1*time.Second, func() (bool, error) {
+		err := manager.WaitForAttachAndMount(context.Background(), pod)
```

Comment:
```suggestion
		return manager.WaitForAttachAndMount(context.Background(), pod) == nil, false
```

Reply:
I think you want to type ``return manager.WaitForAttachAndMount(context.Background(), pod) == nil``
since the return type is (bool, error)